docs/release-notes/cloud/cloud-2022-12.md

```diff
@@ -0,0 +1,80 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud December 2022.
+included_jira_versions: ['2022.Q4.5']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.6.5
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.7.36
+---
+
+# Cloud December 2022
+
+These release notes are for the Codacy Cloud updates during December 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The pull request detail page now displays a message when Codacy can't calculate the pull request metrics. The message includes a link to the coverage report logs to help you [validate your coverage setup](../../coverage-reporter/index.md#validating-coverage). (IO-152)
```

Comment:
The release note says "a link to the coverage report logs", but the screenshot shows "Go to coverage settings". Is the message linking to logs or settings?

Reply:
Good point! The link navigates to the **Coverage** tab in the repository settings (for example, [this page](https://app.codacy.com/gh/codacy/docs/settings/coverage) for the codacy/docs repository).

I was trying to shorten the release note a bit, but we can make it more explicit like this, what do you think?

```suggestion
-   The pull request detail page now displays a message when Codacy can't calculate the pull request metrics. The message includes a link to the **Coverage** tab in your repository settings where you can see the coverage report logs to help you [validate your coverage setup](../../coverage-reporter/index.md#validating-coverage). (IO-152)
```

Reply:
Humm... Maybe I'm missing some context here. 🤔 Where are the report logs? How the users will know what to do when they reach the Coverage tab? It seems that it would be more useful to have the link pointing to that documentation.

Reply:
Before we displayed this message only at the top of the repository pages and indeed it included a link to the docs. But after [COV-46](https://codacy.atlassian.net/browse/COV-46) we changed the link so it navigated directly to the **Coverage** tab in the repository settings. I guess this is relevant feedback for @fjrdomingues and @RaquelHipolito.